config/src/test/java/org/springframework/security/config/annotation/web/configurers/saml2/Saml2LoginConfigurerTests.java

```diff
@@ -16,26 +16,23 @@
 
 package org.springframework.security.config.annotation.web.configurers.saml2;
 
-import java.io.ByteArrayOutputStream;
```

Comment:
In each file that you change, will you please update the copyright year? For example, if it says `2012-2020`, update it to say `2012-2021`.